### PR TITLE
You can now cancel AI intellicard wiping

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: You can now cancel AI intellicard wiping.
/:cl:

In theory, at least. I can't figure out how to compile tgui and there are 0 guides for it
@Mervill @RemieRichards you've done this send help

### CODE REVIEW

**Comment by ChangelingRain** [comment]
@lzimann compiled it for me but I still need to figure out how to do it so I can do other uis!

**Comment by Shadowlight213** [comment]
Is there a reason why it couldn't be stopped before? Just curious.

**Comment by ChangelingRain** [comment]
I don't know if there was one, honestly. It might have been a shitty coder but hell if I know.

File: code/game/objects/items/devices/aicard.dm [hunk 1]
```diff
@@ -62,17 +62,19 @@
 		return
 	switch(action)
 		if("wipe")
-			var/confirm = alert("Are you sure you want to wipe this card's memory? This cannot be undone once started.", name, "Yes", "No")
-			if(confirm == "Yes" && !..())
-				flush = TRUE
-				if(AI && AI.loc == src)
-					AI.suiciding = TRUE
```
Comment (Shadowlight213) [inline code/game/objects/items/devices/aicard.dm]: Was `suiciding` being set to true here important for anything?

Comment (ChangelingRain) [inline code/game/objects/items/devices/aicard.dm]: I checked. Besides fucking up and reporting the AI as a suicide, no.

**Comment by Cyberboss** [comment]
We really need a step by step for compiling tgui

**Comment by RemieRichards** [comment]
the tgui looks fine,
run build_assets.bat
run reload.bat once build_assets finishes

**Comment by ChangelingRain** [comment]
I don't actually know what to install/how to install it so that those work; I tried for most of yesterday with a vague idea of what I needed and got only failure because I was probably doing something wrong.

**Comment by RemieRichards** [comment]
@ChangelingRain install_depedencies.bat

**Comment by ChangelingRain** [comment]
I'm getting weird errors with that what do I need installed for *that* to work

**Comment by optimumtact** [comment]
you need to merge master and recompile the tgui minified file as it's conflicting.

blow away the tgui.js file with the latest upstream master, then just recompile your changes into it.

**Pull request merged**